spring-cloud-generator/generate-library-list.sh

```diff
@@ -64,6 +64,11 @@ for d in ./google-cloud-java/*java-*/; do
     echo "$artifact_id is already present in manual modules."
     continue
   fi
+  # checks if library is in the exclusion list
+  if [[ $(cat excluded_libs.txt | tail -n+2 | grep $artifact_id | wc -l) -ne 0 ]] ; then
+    echo "$artifact_id is in the exclusion list"
+    continue
+  fi
```

Comment:
Can you also populate the [library_list.txt](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/b0846c64d254044fc6f3deab7fff3de6a77f2d73/spring-cloud-generator/library_list.txt) with these removals?
This script is not run within `generate-all.sh` with current setup.

Reply:
Will do

Reply:
Done